### DISCUSSION TITLE
Transmission takes long to establish connections and download

### DISCUSSION THREAD

**Discussion by al3xandru** [transmission/transmission#5813] [General]
I have had this feeling that Transmission is a bit slow in establishing connections and downloading. I've noticed many cases in which Transmission was waiting for sources to download (which I explained as lack of seeds/peers). Then I tried the same downloads in another application. In most of the cases downloads started almost right away and the download speed was higher.

I do understand this is not really backed by anything for now. I just wanted to see if it's purely a subjective observation or if others have observed it or had similar gut feeling maybe figure out what info we could collect to further investigate thi.

**Comment by Pentaphon** [comment]
> I do understand this is not really backed by anything for now. I just wanted to see if it's purely a subjective observation or if others have observed it or had similar gut feeling maybe figure out what info we could collect to further investigate thi.

People have noticed this here https://github.com/transmission/transmission/issues/5261 and there's currently a lot of work being done to try and fix and optimize Transmission that will show up in 4.1.0 that you will see being referenced in that thread. Feel free to help test nightly builds.
